charts/dkube-deployer/templates/config-map.yaml

```diff
@@ -138,8 +138,49 @@ data:
     # E.g: sdb
     STORAGE_CEPH_DISK={{ .Values.optional.storage.cephDisk }}
     [MODELMONITOR]
-    #To enable modelmonitor in dkube. (true / false)
+    # To enable modelmonitor in dkube. (true / false)
     ENABLED={{ .Values.optional.modelmonitor.enabled }}
+
+    [TIMESCALE]
+
+    # To enable promscale in dkube. (true/false)
+    ENABLED={{ .Values.optional.promscale.enabled }}
+
+    BACKUP_ENABLED={{ .Values.optional.promscale.timescaledb.backup.enabled }}
+
+    # Timescale db details needs to provide if you are hosting 
+    # To use an external timescaledb. (true/false).
+    EXTERNAL={{ .Values.optional.promscale.timescaledb.external }}
+    DB={{ .Values.optional.promscale.timescaledb.db }}
+    USER={{ .Values.optional.promscale.timescaledb.user }}
+    HOST={{ .Values.optional.promscale.timescaledb.host }}
+    PASSWORD={{ .Values.optional.promscale.timescaledb.password }}
+    PORT={{ .Values.optional.promscale.timescaledb.port }}
+
+    # S3 details for backup only required if external is false and you want to use 
+    # custom s3 source for backup
+    # default: /timescale
+    S3_PATH={{ .Values.optional.promscale.timescaledb.backup.s3.path }}
+    S3_HOST={{ .Values.optional.promscale.timescaledb.backup.s3.endpoint }}
+    S3_BUCKET={{ .Values.optional.promscale.timescaledb.backup.s3.bucket }}
+    S3_REGION={{ .Values.optional.promscale.timescaledb.backup.s3.endpoint }}
+    S3_KEY={{ .Values.optional.promscale.timescaledb.backup.s3.key }}
+    S3_SECRET_KEY={{ .Values.optional.promscale.timescaledb.backup.s3.secret }}
+
+    [PROMSCALE]
+    # Promscale Configuartion
+    # To enable promscale in dkube. (true/false)
+    ENABLED={{ .Values.optional.promscale.enabled }}
+    # if promscale is hosted external
+    EXTERNAL={{ .Values.optional.promscale.external.enabled }}
+    # Base URL 
+    # https://docs.timescale.com/promscale/latest/send-data/prometheus/#configure-prometheus-to-read-and-write-data-from-promscale       
+    BASE_URL={{ .Values.optional.promsacle.external.base_url }}
+    # Default Value : /read  
+    READ_SUFFIX={{ .Values.optional.promscale.external.read_url_suffix }}
```

Comment:
What are these and how are they used?

Reply:
The like mentioned there has reference for read_url and write_url so instead of user writing both url again he can only specify subpath. /read or /write